source/math/FloatPoint3D.ooc

```diff
@@ -67,6 +68,7 @@ FloatPoint3D: cover {
 	operator > (other: This) -> Bool { this x > other x && this y > other y && this z > other z }
 	operator <= (other: This) -> Bool { this x <= other x && this y <= other y && this z <= other z }
 	operator >= (other: This) -> Bool { this x >= other x && this y >= other y && this z >= other z }
+	toIntPoint3D: func -> IntPoint3D { IntPoint3D new(this x floor() as Int, this y floor() as Int, this z floor() as Int) }
```

Comment:
You're right, the 2D version used it for some reason and it got copied along with the rest. Will fix both.